projects/components/src/lib/form/filter-state-option/filter-state-option.scss

```diff
@@ -1,5 +1,5 @@
 .results-count {
-  margin-top: 8px;
+  margin-top: -38px;
```

Comment:
It's usually a red flag when setting negative margin. Are you sure this was needed?

Reply:
The negative margin was because the results count is not part of the advanced search component, so on the new dashboard edit page, it sits a bit awkwardly under the new button.

<img width="572" alt="Screen Shot 2019-07-10 at 11 36 48 PM" src="https://user-images.githubusercontent.com/11209247/61027730-c1d06900-a36b-11e9-8d3c-00eb3fda33c0.png">